ark/utils/spatial_analysis_utils_test.py

```diff
@@ -144,22 +143,35 @@ def test_compute_close_cell_num():
         dist_mat=example_dist_mat, dist_lim=100, analysis_type="cluster",
         current_fov_data=all_data, cluster_ids=cluster_ids)
 
-    assert example_closenum[0, 0] == 16
-    assert example_closenum[1, 1] == 25
-    assert example_closenum[2, 2] == 1
+    assert example_closenum[0, 0] == 12
+    assert example_closenum[1, 1] == 20
+    assert example_closenum[2, 2] == 0
```

Comment:
We should add a test that checks for asymmetry

Reply:
the regular close-num calculation is still symmetric, it's just the randomization that has the asymmetry.

In particular, cells of lineage `i` are fixed in place while `num_cells_of_linage[j]` cells are randomly selected and checked to be within the distance limit.  Previously, the cells of lineage `i` where also randomized, which, _while informative_, is more reflective of global cell 'topography' than local lineage organization.

Regardless, I can add a check for asymmetry within the `close_num_rand` tests